src/tools/pin/pintool.cpp

```diff
@@ -44,7 +44,11 @@ QString PinTool::description() const {
 
 QWidget* PinTool::widget() {
     PinWidget *w = new PinWidget(m_pixmap);
-    w->setGeometry(m_geometry);
+    QRect adjusted_pos = QRect (m_geometry.left() - LAYOUT_MARGIN,

```

Comment:
Adjust the position of the pinned widget to keep the same position as the captured screenshot.